scripts/update/nginx.sh

```diff
@@ -59,37 +69,45 @@ function update_nginx() {
                 -e "s/;opcache.max_accelerated_files=2000/opcache.max_accelerated_files=4000/" \
                 -e "s/;opcache.revalidate_freq=2/opcache.revalidate_freq=240/" /etc/php/$version/fpm/php.ini
             phpenmod -v $version opcache
+            echo_success "php $version configured"
         fi
     done
 
     if [[ ! -f /etc/nginx/modules-enabled/50-mod-http-fancyindex.conf ]]; then
         mkdir -p /etc/nginx/modules-enabled/
         ln -s /usr/share/nginx/modules-available/mod-http-fancyindex.conf /etc/nginx/modules-enabled/50-mod-http-fancyindex.conf
+        echo_log_only "Did a fancyindex link thing"
     fi
 
     phpversion=$(php_service_version)
 
+    #todo what do these changes all do? can they be wrapped into one if statement or documented?
```

Comment:
If I remember correctly, this first line *is* the check -- whether or not the system's PHP in use matches up with the nginx configs.